docs/pages/desktop-access/getting-started.mdx

```diff
@@ -314,6 +314,62 @@ Computer Configuration > Policies > Administrative Templates > Windows Component
   ![Disable Require](../../img/desktop-access/disable.png)
 </Figure>
 
+### Configure certificate for RDP connections
+
+<Admonition type="note" title="Secure Cipher Suites">
+  Teleport's RDP client supports only secure algorithms
+  for making TLS connections, so we have to configure a Windows server
+  to support those cipher suites as well.
+  This step is only *necessary* for Windows Server 2012 R2 as it does not support
+  secure algorithms by default.
+</Admonition>
+
+In this step we'll create a new certificate template that uses elliptic curve cryptography, and then configure our GPO
+to use the newly created template to issue certificates used for Remote Desktop connections.
+
+## Create a certificate template
+
+In this step we create a certificate template that uses elliptic curve P-384 and uses SHA384 as the signature algorithm.
```

Comment:
```suggestion
In this section, we will create a certificate template that uses elliptic curve P-384 and uses SHA384 as the signature algorithm.
```
To distinguish the beginning of this paragraph from the beginning of the last paragraph. The reader may be wondering, "when does the step begin?"